### DISCUSSION TITLE
How can i pass dynamic state value to date attribute

### DISCUSSION THREAD

**Discussion by JituG22** [ndresx/react-countdown#166] [Q&A]
How can i pass react dynamic state value to Countdown component  date attribute
Ex :
<Countdown autoStart={true} date={new **Date(meetingEndTime.date)**} renderer={renderer}/>

I have pass my react state **meetingEndTime.date** like above code snippet to date attribute but seems like  not work

**Comment by ndresx** [comment]
This should be working as you have it. What doesn't work?
